apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Tests/Tasks/TestAwaiter.cs

```diff
@@ -35,7 +35,16 @@ public TestAwaiter(Task task, TaskScheduler taskScheduler)
         public void UnsafeOnCompleted(Action continuation) =>
             _task.ContinueWith(_ => continuation(), CancellationToken.None, TaskContinuationOptions.DenyChildAttach, _taskScheduler);
         public bool IsCompleted => _task.IsCompleted;
-        public void GetResult() => _task.Wait();
+        public void GetResult()
```

Comment:
Is this now equivalent to our `_task.WaitWithUnwrappedExceptions()` method? (Slightly more efficient when an exception is thrown, admittedly - maybe we should change our implementation.)